### PR TITLE
Adding error log level

### PR DESCRIPTION
**Pull request by jcerrada**
This change allow us to know whether an event comes from an error or not and log it accordingly.

### CODE REVIEW

File: Plugin/ELK/Domain/Event/DomainEventSubscriber.php [hunk 1]
```diff
@@ -144,16 +147,18 @@ public function handle(DomainEventWithRepositoryReference $domainEventWithReposi
         try {
             $this
                 ->getLogger()
-                ->info(
+                ->addRecord(
+                    $event instanceof ExceptionWasCached ? Logger::ERROR : Logger::INFO,
                     json_encode([
+                            'environment' => getenv('APP_ENV') ?g? 'prod',
```
Comment (hkulekci) [inline Plugin/ELK/Domain/Event/DomainEventSubscriber.php]: `'environment' => getenv('APP_ENV') ?? 'prod'`

Comment (hkulekci) [reply Plugin/ELK/Domain/Event/DomainEventSubscriber.php]: You had added `g` character between question marks.

**Comment by mmoreram** [comment]
Ping @jcerrada

**Comment by mmoreram** [comment]
The issue has been done and tested in https://github.com/apisearch-io/search-server/pull/213. Closing.

**Pull request closed without merging**